### PR TITLE
New cpp codebytes

### LINKED ISSUE #2155: [Edit] C++ Add Codebytes
### Type of Edit (select all that apply)

Add new content (definitions, codeblocks, etc.)

### Description (optional)

Please add Codebytes to the listed entries.       
https://www.codecademy.com/resources/docs/cpp/math-functions/abs
https://www.codecademy.com/resources/docs/cpp/math-functions/acos
https://www.codecademy.com/resources/docs/cpp/math-functions/asin

Bear in mind the following two principles:
1) There should only be one Codebyte per entry listed as `## Codebyte Example` at the end of the entry.
2) The codebytes should be new examples, not copies of the prior examples.

Please refer to the [entry templates](https://github.com/Codecademy/docs/blob/main/documentation/), [content standards](https://github.com/Codecademy/docs/blob/main/documentation/content-standards.md), and [style guide](https://github.com/Codecademy/docs/blob/main/documentation/style-guide.md) when working on the PR for this issue.

### Code of Conduct

- [X] By submitting this issue, I agree to follow Codecademy Doc's [Code of Conduct](https://github.com/Codecademy/docs/blob/main/.github/CODE_OF_CONDUCT.md).

### For Maintainers

- [x] Labels added
- [x] Issue is assigned

### PR DESCRIPTION
**Pull request by Luke943**
### Description

- Codebyte examples have been added for three C++ functions: abs, acos, asin.
- Closes #2155

### Type of Change

- Updating the documentation

### Checklist

- [x] All writings are my own.
- [x] My entry follows the Codecademy Docs style guide.
- [x] My changes generate no new warnings.
- [x] I have performed a self-review of my own writing and code.
- [x] I have checked my entry and corrected any misspellings.
- [x] I have made corresponding changes to the documentation if needed.
- [x] I have confirmed my changes are not being pushed from my forked `main` branch.
- [x] I have confirmed that I'm pushing from a new branch named after the changes I'm making.
- [x] Under "Development" on the right, I have linked any issues that are relevant to this PR (write "Closes #<issue number> in the "Description" above).

### CODE REVIEW

File: content/cpp/concepts/math-functions/terms/acos/acos.md [hunk 1]
```diff
@@ -37,3 +37,21 @@ int main() {
...
 }
 ```
+
+## Codebyte Example
+
+The following example is runnable and outputs the inverse cosine of `-0.5`:
+
+```codebyte/cpp
+#include <iostream>
+#include <cmath>
```
Comment (yangc95) [inline content/cpp/concepts/math-functions/terms/acos/acos.md:47]: may want to include std here and for other examples so we don't have to keep repeating std

**Review by yangc95 — CHANGES_REQUESTED** [review]
Hey @Luke943, just one main comment here for all three codebytes

**Comment by Luke943** [comment]
Thanks @yangc95, I've made the change to the files.

**Review by yangc95 — APPROVED** [review]
@Luke943 perf placement! Thanks for making those changes, looks good for another quick review

**Review by caupolicandiaz — APPROVED** [review]
@Luke943 @yangc95 LGTM

**Comment by Luke943** [comment]
Great! Thanks both @yangc95 @caupolicandiaz

**Pull request merged**